app/templates/components/post-search-results-affiliate-item.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="post-search-results-affiliate-item">
+  {{#if affiliateUnit.image}}
+    <img class="post-search-results-affiliate-item__image" src={{affiliateUnit.image}} alt={{affiliateUnit.heading}}>
+  {{/if}}
+  <a
+    href={{{affiliateUnit.link}}}
+    class="post-search-results-affiliate-item__title wds-leading-tight"
+    onclick={{action "trackClick" postIndex}}
+  >
+    {{affiliateUnit.heading}}
+  </a>
+  <div class="post-search-results-affiliate-item__details">
+      Watch it on Disney+
```

Comment:
Actually this will come from `affiliateUnit.subheading`

Reply:
Ahh that makes sense.

Reply:
https://github.com/Wikia/mobile-wiki/pull/1808/commits/012427775fd963d54ad2e11aa0f49b61977945d8